lib/src/tools/display/display_service_contract.dart

```diff
@@ -0,0 +1,53 @@
+abstract class DisplayService {
+  ///
+  /// Call this method to set brightness for one display.
+  ///
+  void setBrightness(DisplayDevice display, double brightness);
+
+  /// Gets whether Display Dark Mode is turn ON.
```

Comment:
Please use three lines for comment (to follow style)

```
///
/// Gets whether Display Dark Mode is turn ON.
///
```

Reply:
Added  three lines for comment.